src/visualization/include/iDynTree/Visualizer.h

```diff
@@ -437,8 +450,16 @@ class IModelVisualization
      * Get a reference to the internal IJetsVisualization interface.
      */
     virtual IJetsVisualization& jets() = 0;
-
-
+        
+    /**
+     * Get the transformation of the model (root link) with respect to visualizer world
+     */
+    virtual Transform getWorldToModelTransform() = 0;
```

Comment:
Can you document which transform this method is returning using the notation \f$ world_H_root \f$ notation?

Reply:
I also suggest to avoid the `WorldToModel` or `WorldToLink` form, that I think it may be misleading. If I understand correctly, this methods are returning world_H_root, that is the transform that if multiplied on the right for a homegenous vector, it transform it from the root link frame **to** the world (so I would say that it is a ModelToWorld). Given that this is misleading, I would just call it `getWorldModelTransform` and `getWorldLinkTransform`.